Source/WebCore/Modules/mediastream/gstreamer/GStreamerWebRTCUtils.cpp

```diff
@@ -213,7 +213,7 @@ RTCRtpSendParameters toRTCRtpSendParameters(const GstStructure* rtcParameters)
 
     RTCRtpSendParameters parameters;
     if (auto transactionId = gstStructureGetString(rtcParameters, "transaction-id"_s))
-        parameters.transactionId = transactionId.toString();
+        parameters.transactionId = String(transactionId.span());
```

Comment:
Why do we have to explicitly write `String()` here? Doesn't it compile without it?